less/events.less

```diff
@@ -100,4 +126,44 @@
         background: none;
         border: 1px solid #FFFFFF;
     }
-}
\ No newline at end of file
+}
+
+.calendarIcon {
+    color: #3E7B96;
+    margin-right: 5px;
+}
+
+.calendarIcon-ve {
+    color:  #FFFFFF !important;
+}
+
+/** CustomSearch Style start **/
+/* enable absolute positioning */
+.inner-addon { 
+    position: relative; 
+}
+
+/* style icon */
+.inner-addon .fa-search {
+  position: absolute;
+  padding: 10px;
+  pointer-events: none;
+}
+
+/* align icon */
+.left-addon .fa-search { 
```

Comment:
Try using more nesting for stuff like this.
You'll be able to put the code from line 162 in the same nest and it will be easier to read.